cloud/pulsar-admin/README.md

```diff
@@ -22,3 +24,29 @@ The `pulsar-admin` supports to connect to Pulsar cluster through Token, as shown
     tenants list
 ```
 
+Output:
+
+```text
+"public"
+"pulsar"
+```
+
+## OAuth2
+
+The `pulsar-admin` supports to connect to Pulsar cluster through OAuth2, as shown below:
+
+```shell script
+bin/pulsar-admin --admin-url https://cloud.streamnative.dev:443 \
+  --auth-plugin org.apache.pulsar.client.impl.auth.oauth2.AuthenticationOAuth2 \
+  --auth-params '{"privateKey":"file:///path/to/key/file.txt",
```

Comment:
file.txt is a json format right?  how about change it into file.json?  If changing this, please also change similar places.